### ISSUE TITLE
Deploy Aave v3 Borrow strategy to vaETH

### ISSUE THREAD

**Issue by green-jeff** [vesperfi/vesper-pool-ops#231]
Aave V3 is now live on mainnet and offers better net rates on ETH supply and stablecoin borrows.

Deploy Strategy of: Deposit ETH -> Borrow DAI -> deposit to Vesper

**Comment by patidarmanoj10** [comment]
@virendrapatidar any update on this?

**Comment by virendrapatidar** [comment]
@patidarmanoj10 - Didn't start working on it yet. 
Let me prepare the config and unit test for it first.

**Comment by virendrapatidar** [comment]
@patidarmanoj10 

Configuration added, Unit tests are okay. I will deploy once PR is approved.

**Comment by virendrapatidar** [comment]
Nonce 968 proposed for Aave v3 Borrow strategy deployment for vaETH pool.

**Comment by virendrapatidar** [comment]
Executed 968 txn.

**Comment by virendrapatidar** [comment]
Closing this ticket. Need another ticket for the weight update.
